apps/air-discount-scheme/backend/src/app/modules/discount/discount.service.ts

```diff
@@ -71,13 +72,13 @@ export class DiscountService {
     value: T,
     ttl: number = ONE_DAY,
   ): Promise<void> {
-    return this.cacheManager.set(key, value, { ttl })
+    return this.cacheManager.set(key, value, ttl * 1000)
```

Comment:
nitnit: Since 1000 is used on multiple occasions maybe store in a const

Reply:
Agree to disagree 😇